lamp-small-rc.yaml

```diff
@@ -49,6 +49,8 @@ parameters:
     constraints:
     - allowed_pattern: "^[a-zA-Z0-9.-_]{1,255}$"
       description: Must only contain alphanumeric characters, '-', '_', and '.'
+    - length: { min: 1, max: 16 }
+      description: User name must be between 6 and 8 characters
```

Comment:
This will not do what you expect per the length constraint above, see https://github.com/rackspace-orchestration-templates/lamp-small/pull/3 for my equivalent comment.